komodo/ask_for_tags.py

```diff
@@ -0,0 +1,218 @@
+#!/usr/bin/env python3
+
+import base64
+import os
+import re
+from collections import defaultdict
+from textwrap import dedent
+from typing import Callable, List, Mapping
+
+import yaml
+from github import Github
+from github.Commit import Commit
+from github.GithubException import GithubException, UnknownObjectException
+from github.Repository import Repository
+
+not_in_komodo_releases_project_msg = (
+    "We expect this script to be run in the komodo-releases project"
+)
+
+
+def get_packages_that_use_main() -> List[str]:
+    bleeding_release_file = "releases/matrices/bleeding.yml"
+    duplicate_pkgs = ["everest-models"]  # this is included in the everest repo
```

Comment:
In principle, `komodo` is a general purpose FOSS tool. I wonder if it's possible to write the functions in this module in such a way that they do not assume it's only being run by Equinor? I will mark some other lines that can perhaps be refactored.

Reply:
(Just to add: another possibility is that this project is not meant for general consumption at all, which is totally fine -- but then we should either close it or make it clear that it's an Equinor tool.)